list/mon_lib_list.cif

```diff
@@ -42103,6 +42119,69 @@ _chem_mod_tor.new_value_angle
 _chem_mod_tor.new_value_angle_esd
 MPRm1 delete C2 C3 S3 HS3 . .
 
+data_mod_LYZm1
+
+loop_
+_chem_mod_atom.mod_id
+_chem_mod_atom.function
+_chem_mod_atom.atom_id
+_chem_mod_atom.new_atom_id
+_chem_mod_atom.new_type_symbol
+_chem_mod_atom.new_type_energy
+_chem_mod_atom.new_charge
+LYZm1          delete         HZ2       .         H         H         0
+LYZm1          delete         HZ3       .         H         H         0
+LYZm1          change         NZ        .         N         N31       0
+
+loop_
+_chem_mod_bond.mod_id
+_chem_mod_bond.function
+_chem_mod_bond.atom_id_1
+_chem_mod_bond.atom_id_2
+_chem_mod_bond.new_type
+_chem_mod_bond.new_value_dist
+_chem_mod_bond.new_value_dist_esd
+_chem_mod_bond.new_value_dist_nucleus
+_chem_mod_bond.new_value_dist_nucleus_esd
+LYZm1          delete         NZ        HZ2       single         .              .         .         .
+LYZm1          delete         NZ        HZ3       single         .              .         .         .
+LYZm1          change         CD        CE        single         1.519          0.0197    1.519          0.0197
+LYZm1          change         CE        NZ        single         1.451          0.0100    1.451          0.0100
+LYZm1          change         NZ        HZ1       single         0.868          0.0200    1.016          0.0100
+
+loop_
+_chem_mod_angle.mod_id
+_chem_mod_angle.function
+_chem_mod_angle.atom_id_1
+_chem_mod_angle.atom_id_2
+_chem_mod_angle.atom_id_3
+_chem_mod_angle.new_value_angle
+_chem_mod_angle.new_value_angle_esd
+LYZm1          delete         CE        NZ        HZ2       .              .
+LYZm1          delete         CE        NZ        HZ3       .              .
+LYZm1          delete         HZ1       NZ        HZ2       .              .
+LYZm1          delete         HZ1       NZ        HZ3       .              .
+LYZm1          delete         HZ2       NZ        HZ3       .              .
+LYZm1          change         CG        CD        CE        110.882        1.50
+LYZm1          change         CE        CD        OH        110.735        1.50
+LYZm1          change         CD        CE        NZ        111.845        3.00
+LYZm1          change         CD        CE        HE3       108.828        1.50
+LYZm1          change         CD        CE        HE2       108.828        1.50
+LYZm1          change         CE        NZ        HZ1       115.193        3.00
+
+loop_
+_chem_mod_tor.mod_id
+_chem_mod_tor.function
+_chem_mod_tor.atom_id_1
+_chem_mod_tor.atom_id_2
+_chem_mod_tor.atom_id_3
+_chem_mod_tor.atom_id_4
+_chem_mod_tor.id
+_chem_mod_tor.new_value_angle
+_chem_mod_tor.new_value_angle_esd
+_chem_mod_tor.new_period
+LYZm1          delete         CD        CE        NZ        HZ3       .           .           .           .
```

Comment:
Current gemmi (0.6.0) does not allow "." for _chem_mod_tor.new_period, that's why the test failed. Upcoming 0.6.1 will be ok with it.

Reply:
Ok. I see. I need to use new acedrg.